opentelekomcloud/common/tags.go

```diff
@@ -9,9 +9,9 @@ import (
 // TagsSchema returns the schema to use for tags.
 func TagsSchema() *schema.Schema {
 	return &schema.Schema{
-		Type:         schema.TypeMap,
-		Optional:     true,
-		ValidateFunc: ValidateTags,
+		Type:     schema.TypeMap,
+		Optional: true,
+		Elem:     &schema.Schema{Type: schema.TypeString},
```

Comment:
Most of the tag validation use common standard validation rules, some of them like CBR vaults may be different.

With this validation terraform can tell that configuration is invalid without launching configuration itself, which saves a lot of time in case configuration has lots and lots of resources.

That's why we need validation inside schema.
Please restore all validation fields.

Reply:
I've found a few cases that don't pass the verification, but I've restored them as you requested.